cmd/kpod/ps.go

```diff
@@ -395,7 +395,9 @@ func getJSONOutput(containers []*libkpod.ContainerData, nSpace bool) (psOutput [
 
 func generatePsOutput(containers []*libkpod.ContainerData, server *libkpod.ContainerServer, opts psOptions) error {
 	containersOutput := getContainers(containers, opts)
-	if len(containersOutput) == 0 {
+	// In the case of JSON, we want to continue so we at least pass
+	// valid JSON to the consumer
```

Comment:
micro nit for consideration.   "least pass" to may "least pass {} which is"

Reply:
fixed.